### PR TITLE
Add CoyIM.app v0.1

### PR DESCRIPTION
**Pull request by sriprasanna**

### CODE REVIEW

File: Casks/coyim.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'coyim' do
+  version '0.1'
+  sha256 'f7034d71ef81c116bbc59a784b65f835e1dac0276be02e83d7b2ed534551aceb'
+
+  url "https://dl.bintray.com/twstrike/coyim/v#{version}/mac-bundle/coyim.dmg"
```
Comment (vitorgalvao) [inline Casks/coyim.rb]: Needs a `# bintray.com is the official download host per the vendor homepage` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/cb487ad405f74cb1f26b93e3f3e8412c82a22e34/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/7c010b42bd7ac679c1f002a0368b36bb3f495612/doc/CASK_LANGUAGE_REFERENCE.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

Comment (sriprasanna) [inline Casks/coyim.rb]: Will do, thanks.

**Comment by vitorgalvao** [comment]
Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by vitorgalvao** [comment]
On hold while [waiting for clarification](https://github.com/twstrike/coyim/issues/178#issuecomment-168817393) from @tdruiva.

**Comment by sriprasanna** [comment]
Squashed my commits into single one.

**Comment by vitorgalvao** [comment]
[Merging](https://github.com/twstrike/coyim/issues/178#issuecomment-168831720).

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It was merged directly as 0d154ae0b3e551ef29c43a0d3ec0c594091e67b8 to keep commit history cleaner. Your contribution is still credited to you.

**Comment by sriprasanna** [comment]
Thanks!

**Pull request closed without merging**